tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -33,23 +32,24 @@ class SbtServerSuite
 
   test("too-old") {
     cleanWorkspace()
+    writeLayout(
+      s"""|/project/build.properties
+          |sbt.version=$preBspVersion
+          |/build.sbt
+          |${SbtBuildLayout.commonSbtSettings}
+          |scalaVersion := "${V.scala212}"
+          |""".stripMargin
+    )
     for {
-      _ <- server.initialize(
-        s"""|/project/build.properties
-            |sbt.version=$preBspVersion
-            |/build.sbt
-            |scalaVersion := "${V.scala212}"
-            |""".stripMargin
-      )
+      _ <- server.initialize()
+      _ <- server.initialized()
       _ = assertNoDiff(
         client.workspaceMessageRequests,
         List(
-          importBuildMessage,
-          progressMessage
```

Comment:
Why did the progressMessage disappear ? It's better, I am just asking out of curiosity.

Reply:
In `TestingClient` the default response of the `importBuildMessage` is:
```scala
var importBuild: MessageActionItem = ImportBuild.notNow
```

So Metals does not run `bloopInstall` and that's why there is no `progressMessage` anymore. Instead it switches to sbt as the build server.